app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -38,9 +38,9 @@ public function panel(Panel $panel): Panel
             ->favicon(config('app.favicon', '/pelican.ico'))
             ->topNavigation(config('panel.filament.top-navigation', true))
             ->maxContentWidth(config('panel.filament.display-width', 'screen-2xl'))
-            ->profile(EditProfile::class, false)
             ->login(Login::class)
             ->userMenuItems([
+                'profile' => MenuItem::make()->label('Profile')->url(fn () => EditProfile::getUrl(panel: 'app')),
```

Comment:
Use translation from filament.

```suggestion
                'profile' => MenuItem::make()->label(trans('filament-panels::pages/auth/edit-profile.label'))->url(fn () => EditProfile::getUrl(panel: 'app')),
```